boards/xtensa/esp32s2/common/scripts/esp32s2_sections.ld

```diff
@@ -1,5 +1,5 @@
 /****************************************************************************
- * boards/xtensa/esp32s2/esp32s2-saola-1/scripts/esp32s2_flash.ld
+ * boards/xtensa/esp32s2/common/scripts/esp32s2_sections.ld
```

Comment:
Since the `esp32s2_iram.ld` and `esp32s2_flash.ld` scripts have been merged into one, the following adjustments are now required:
- `.flash.rodata` output section shall be allocated into `default_rodata_seg`.
- `.flash.text` output section shall be allocated into `default_code_seg`.

Reply:
Fixed.